sentry/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM sentry:9.0.0
+
+LABEL name=sentry version=$SENTRY_VERSION \
```

Comment:
`$SENTRY_VERSION` is set on the instruction right below

Reply:
I wanted to :sonic: fix it but failed :(